### ISSUE TITLE
Domain and Class confusion

### ISSUE THREAD

**Issue by philbarker** [t3-innovation-network/desm#341]
We should try to be less confusing about domains and classes. Difficult as both terms have several meanings.

One things that would help would be to keep Domain as a term for describing the schema-independent name of a type of thing that properties are about, so the Domain would be defined by an Abstract Class, there would be one spine per domain. The domain mapping  that is done when a mapper uploads their schema then involves them selecting classes from their schema that are relevant to a given domain.

Most standards, I think, have classes rather than domains.

If we agree on that, then this pop up, which appears when the schema file is uploaded and processed, needs changing to talk about classes, not domains.
![image](https://user-images.githubusercontent.com/658047/201925313-f4ff0987-3426-45bd-9bce-4b23b4de6cf8.png)

Also the blue pop-up that appears when the selected domains/classes are processed should say "You selected the following classes":
![image](https://user-images.githubusercontent.com/658047/201926299-1dbf2590-0654-44ee-8154-07afa15231bd.png)

**Comment by philbarker** [comment]
We also through "Category" into the mix when we get to the view mappings screen:
![image](https://user-images.githubusercontent.com/658047/206536829-011a1c75-c832-44cd-af8a-fab4f05bca87.png)

**Comment by philbarker** [comment]
We should review the documentation so that it makes better sense to the end users, this will be only one of the things addressed.

**Comment by dlawsonQIP** [comment]
Let's discuss this as a group....some of the changes that need to be addressed require modifications within DESM (such as terminology used in the error messages) in addition to clarification in the documentation. It will be easier to revise the documentation once the changes to the DESM user interface have been made.

**Comment by philbarker** [comment]
Yes, let's discuss. I have moved this to priority 2 to reflect that it needs to wait until some other work is done.

**Comment by philbarker** [comment]
@dlawsonQIP @AH1CEDS  Go through whole mapping process, look at interface and documentation and see what is confusing,

**Comment by philbarker** [comment]
Closing this as it should be covered by issue #600 and other UI/UX issues, and addressed in the documentation when the new UI is documented.
